src/utils/string.ts

```diff
@@ -0,0 +1,31 @@
+import { DonorResponse } from '@/types/donors';
+
+// Perform a set of replacements on a string
+export function replaceMultiple(
+  str: string,
+  replacements: Map<string, string>
+): string {
+  let result: string = str;
+  replacements.forEach((replaceValue, searchValue) => {
+    result = result.replace(searchValue, replaceValue);
```

Comment:
This will only replace the first instance of `searchValue`. Can it be updated to replace all instances?